docs/contributor/draft/rate-limit-design.md

```diff
@@ -0,0 +1,80 @@
+# Rate Limit Proposal
+
+## Spec 
+
+| field                                            | type        | description                                                     | required                      |
+|--------------------------------------------------|-------------|-----------------------------------------------------------------|-------------------------------|
+| wokloadSelector                                  | object      | when skipped, it's applied for the whole namespace workloads, if empty and in istio-system ns, applies to all workloads in all namespaces | no                            |
+| workloadSelector.labels                          | object      | allows to select given workload by a label selector                   | no                            |
+| local                                   | object      | allows to describe local rate limit properties                  | yes                           |
+| local.bucket                            | object      | allows to set rate limit capacity through bucket concept        | yes                           |
+| local.bucket.max_tokens                 | int         | max tokens that can be gathered for the rate limit capacity     | yes                           |
+| local.bucket.tokens_per_fill            | int         | number of tokens added per fill cycle                           | yes                           |
+| local.bucket.fill_interval              | time        | interval on which tokens are filled                             | yes                           |
```

Comment:
I'm not sure if it should be duration or time? I remember that we used `duration` in APIRule v1beta1 for CORS maxAge.
```suggestion
| local.bucket.fill_interval              | duration        | interval on which tokens are filled                             | yes                           |
```

Reply:
done